codice/ddf
# DDF-4106 Adds react wrappers over marionette inputs for ease of use

#### What does this PR do?
 - Adds a number of react wrappers for inputs to allow easier usage from React.

#### Who is reviewing it? 
@djblue 
@brianfelix 
@adimka 
@gjvera 

#### How should this be tested?
Check out the dev guide, that's where most of these are available to test.  I tried out replacing one instance where we were using marionette region container with a property view in workspaces-templates component with the text wrapper and it worked well.

#### Any background context you want to provide?
The marionette region container is a nice way to embed any sort of marionette view, but this makes a number of wrappers that hide the fact that you're interacting with a marionette view (almost) entirely.  It also exposes a typed api to make interacting with common components much simpler.

#### What are the relevant tickets?
[DDF-4106](https://codice.atlassian.net/browse/DDF-4106)
